downstream/modules/aap-hardening/proc-file-systems-mounted-noexec.adoc

```diff
@@ -7,13 +7,20 @@
 
 [role="_abstract"]
 
-The Red Hat Enterprise Linux (RHEL) 8 STIG requires that a number of file systems are mounted with the "noexec" option to prevent execution of binaries located in these file systems. The {PlatformNameShort} (AAP) installer runs a preflight check that will fail if any of the following file systems are mounted with the "noexec" option:
+The Red Hat Enterprise Linux 8 STIG requires that a number of file systems are mounted with the "noexec" option to prevent execution of binaries located in these file systems. The {PlatformNameShort} installer runs a preflight check that will fail if any of the following file systems are mounted with the "noexec" option:
```

Comment:
Should noexec be enclosed in single ticks instead of quotes to render as a code phrase?